Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/patientDiscovery.xhtml

```diff
@@ -256,86 +256,76 @@
                     </div>
                 </p:tabView>           
                 <div  class="modal-copyright-prime">
-                    <p:dialog header="Document Metadata" widgetVar="docMetadataWindow" id="documentMetadataDialog"  width="700px" showEffect="clip" hideEffect="clip" modal="true">
+                    <p:dialog header="#{patientSearchBean.selectedCurrentDocument.documentType} for #{patientSearchBean.patientList[0].name} Created by #{patientSearchBean.selectedCurrentDocument.authorPerson} on #{patientSearchBean.selectedCurrentDocument.creationTimeUiDisplay}" widgetVar="docMetadataWindow" id="documentMetadataDialog"  width="700px" showEffect="clip" hideEffect="clip" modal="true">
```

Comment:
can't we have one method for the dialog header.  Seems like a mess of variables to add to a single JSF display.  Also, can handle if something is missing from those variables in the method.